files/src/utils/test-helpers/test-helper.ts

```diff
@@ -0,0 +1,7 @@
+import { setApp, start } from '@glimmer/test-helpers';
+import App from '../../main';
+
+QUnit.config.autostart = false;
```

Comment:
It won't be changed in QUnit itself (for background, see https://github.com/qunitjs/qunit/pull/971#issuecomment-201704792). Don't we need some sort of wrapper/setup to include QUnit in the build anyway?